bluesky/utils.py

```diff
@@ -787,3 +788,33 @@ def sanitize_np(val):
             return val.item()
         return val.tolist()
     return val
+
+
+def status_progress_bar(*status_objects, loop):
+    from tqdm import tqdm
+    N = 2  # max decades to divide steps into
+    pbars = {}
+    for st in status_objects:
+        pbar = tqdm()
+        if hasattr(st, 'device'):
+            initial = status.device.position
```

Comment:
Good point. Let's see how well this works before we worry about a coordinated PR with ophyd. There might be yet other things we want to add to Status.